int64                   id
geometry_msgs/Point     position
float32                 standard_deviation
float32                 weight
geometry_msgs/Twist     velocity
string[]                required_sensor